docs/plugin-protocol/tfplugin5.7.proto

```diff
@@ -629,21 +629,21 @@ message OpenEphemeralResource {
     message Request {
         string type_name = 1;
         DynamicValue config = 2;
+        ClientCapabilities client_capabilities = 3;
     }
     message Response {
         repeated Diagnostic diagnostics = 1;
         optional google.protobuf.Timestamp renew_at = 2;
-        bool is_closable = 3;
-        DynamicValue state = 4;
-        optional bytes private = 5;
+        DynamicValue result = 3;
+        optional bytes private = 4;
+        Deferred deferred = 5;
```

Comment:
I will match the provider side with the new field/naming once this gets merged 👍🏻 

For the removal of `is_closable`, is the plan for `Close` to be called for all ephemeral resources regardless of if they were renewed/not renewed?

Additionally, will returned diagnostics from `Close` stop execution or will they behave more like the `Stop` RPC for providers and just log?

Reply:
Oh yeah, diagnostics from close would only be able to pass on some info to the user, and make Terraform exit with an error. The close has to happen after all dependencies have used the value, so if there were an error there are no more oeprations to stop execution of.